src/content/docs/fr/guides/markdown-content.mdx

```diff
@@ -660,30 +404,91 @@ export default defineConfig({
 Si vous choisissez d'utiliser Prism, Astro appliquera les classes CSS de Prism à la place. Notez que **vous devez apporter votre propre feuille de style CSS** pour que la coloration syntaxique apparaisse !
 
 <Steps>
-1. Choisissez une feuille de style prédéfinie parmi les [Thèmes Prism](https://github.com/PrismJS/prism-themes) disponibles.
-  
-2. Ajouter cette feuille de styles dans [le répertoire `public/` de votre projet](/fr/basics/project-structure/#public).
-  
-3. Chargez-la dans la balise `<head>` de votre page dans un [composant de feuille de style](/fr/basics/layouts/) via une balise `<link>`. (Voir [utilisation basique de Prism](https://prismjs.com/#basic-usage).)
+  1. Choisissez une feuille de style prédéfinie parmi les [Thèmes Prism](https://github.com/PrismJS/prism-themes) disponibles .
+
+  2. Ajoutez cette feuille de style dans [le répertoire `public/` de votre projet](/fr/basics/project-structure/#public).
+
+  3. Chargez-la dans le `<head>` de votre page dans un [composant de mise en page](/fr/basics/layouts/) via une balise `<link>`. (Voir l'[utilisation de base du prisme](https://prismjs.com/#basic-usage).)
 </Steps>
 
-Vous pouvez aussi visiter la [liste des langages supportés par Prism](https://prismjs.com/#supported-languages) pour les options et leur usage.
+Vous pouvez également consulter la [liste des langues supportées par Prism](https://prismjs.com/#supported-languages) pour les options et l'utilisation.
```

Comment:
In computer science we use `langage` and not `langue`. I also reworded a bit to ease the reading:
```suggestion
Vous pouvez également consulter la [liste des langages supportés par Prism](https://prismjs.com/#supported-languages) pour en savoir plus sur ses options et son utilisation.
```